Mathlib/Algebra/Order/Monovary.lean

```diff
@@ -74,6 +83,8 @@ lemma antivaryOn_inv_right : AntivaryOn f g⁻¹ s ↔ MonovaryOn f g s := by
 @[to_additive] alias ⟨Monovary.of_inv, Monovary.inv⟩ := monovary_inv
 @[to_additive] alias ⟨Antivary.of_inv, Antivary.inv⟩ := antivary_inv
 
+omit [CommGroup β] [IsOrderedMonoid β]
```

Comment:
This is fine but it looks to me like you could have not even introduced these typeclass assumptions at all in this section!